requirements-dev.txt

```
@@ -4,6 +4,7 @@ flake8
 flake8-docstrings
 isort
 matplotlib
+mne[data]
```

I prefer to be explicit here:

```suggestion
tqdm
pooch
```

---

Ah, okay. I figured `mne[data]` would future-proof against future MNE changes to data downloading, but being explicit manse sense.